Bottom: 9552ab53452185fa2e6f9b2f76ff7c8713d3d668
Top:    cdc03a2b282cdd6bfe671cde21b348f9eca51db6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-08 22:24:28 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
